test/e2e/test_acceptance.rb

```diff
@@ -0,0 +1,261 @@
+require 'test/unit'
+require 'fluent/test'
+require 'fluent/test/helpers'
+require 'fluent/test/driver/output'
+require 'open3'
+require_relative '../helper'
+
+ETC_DIR = './fluentd/etc'
+
+class TestFluentAcceptance < Test::Unit::TestCase
+  include Fluent::Test::Helpers
+  include TestHelperModule
+
+  def setup
+    super
+    Fluent::Test.setup
+
+    generate_fluent_conf('syslog-json', 'stdout')
+    _, @stdout, _, @fluent_thread = Open3.popen3("fluentd -c #{ETC_DIR}/fluent.conf -p ./fluentd/plugins")
+    while line = @stdout.gets do
+      break if line.include?('is now running worker')
+    end
+
+    next_line_output_message
+    next_line_output_message
+    next_line_output_message
```

Comment:
Create a new function that captures the idea of this: clearing the buffer so you can start reading "new" stuff